assets/styles/css/styles.scss

```diff
@@ -0,0 +1,129 @@
+@import url("https://fonts.googleapis.com/css2?family=Raleway:wght@500;800;900&display=swap");
+@import "../css/nullstyle.css";
+
+@mixin raleway($size) {
+  font-family: Raleway;
+  font-weight: $size;
+}
+
+.raleway-medium {
+  font-family: Raleway;
+  font-weight: 500;
```

Comment:
Why you do not use @mixin raleway here? But do you need realy use font-family in several classes? It would probably be better to use one rule in the parent class